src/scripts/main.js

```diff
@@ -1,6 +1,8 @@
 import { LoginForm } from "./auth/LoginForm.js"
-import { RegisterForm } from "./auth/RegisterForm.js"
-import { Nutshell } from "./Nutshell.js"
+//import { RegisterForm } from "./auth/RegisterForm.js"
```

Comment:
can you uncomment the Register and Nutshell imports, and move both the eventList and eventForm imports to `Nutshell.js` and then just call them there? That way the event stuff should appear right when a user logs in